.github/workflows/build-frontend-canister.yml

```
@@ -0,0 +1,32 @@
+name: Check frontend canister build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  frontend:
+    runs-on: ubuntu-latest
```

Suggestion: specify a name here, that is similar to the other required status names, to make it easy to specify
```suggestion
    runs-on: ubuntu-latest
    name: frontend-canister-up-to-date:required
```